src/doc/book/primitive-types.md

```diff
@@ -171,12 +171,10 @@ and a length.
 ## Slicing syntax
 
 You can use a combo of `&` and `[]` to create a slice from various things. The
-`&` indicates that slices are similar to [references], which we will cover in
+`&` indicates that slices are similar to [references][references], which we will cover in
```

Comment:
I found that it was the [trpl-ebook](https://github.com/killercup/trpl-ebook) that could not tell the `[references]` style. If I use Pandoc to convert the articles to PDF or Tex format, it will get the right result regardless `[references]`, `[references][]` or `[references][references]`.